data/storage/postgres/create_tariffplan_tables.sql

```diff
@@ -486,25 +486,28 @@ CREATE TABLE tp_filters (
   -- Table structure for table `tp_lcr`
   --
 
-  DROP TABLE IF EXISTS tp_lcr;
-  CREATE TABLE tp_lcr (
+  DROP TABLE IF EXISTS tp_suppliers;
+  CREATE TABLE tp_suppliers (
     "pk" SERIAL PRIMARY KEY,
     "tpid" varchar(64) NOT NULL,
     "tenant"varchar(64) NOT NULL,
     "id" varchar(64) NOT NULL,
     "filter_ids" varchar(64) NOT NULL,
     "activation_interval" varchar(64) NOT NULL,
-    "strategy" varchar(32) NOT NULL,
-    "strategy_params" varchar(64) NOT NULL,
+    "sorting" varchar(32) NOT NULL,
+    "sorting_params" varchar(64) NOT NULL,
     "supplier_id" varchar(32) NOT NULL,
-    "ratingplan_ids" varchar(64) NOT NULL,
     "supplier_filter_ids" varchar(64) NOT NULL,
+    "supplier_ratingplan_ids" varchar(64) NOT NULL,
+    "supplier_resource_ids" varchar(64) NOT NULL,
+    "supplier_stat_ids" varchar(64) NOT NULL,
     "supplier_weight" decimal(8,2) NOT NULL,
+    "blocker" BOOLEAN NOT NULL,
     "weight" decimal(8,2) NOT NULL,
     "created_at" TIMESTAMP WITH TIME ZONE
   );
- CREATE INDEX tp_lcr_idx ON tp_lcr (tpid);
- CREATE INDEX tp_lcr_unique ON tp_lcr  ("tpid",  "tenant", "id", "filter_ids");
+ CREATE INDEX tp_suppliers_idx ON tp_suppliers (tpid);
+ CREATE INDEX tp_suppliers_unique ON tp_suppliers  ("tpid",  "tenant", "id", "filter_ids","supplier_id");
```

Comment:
Unique key should also have rating_plan_ids, resource_ids and stat_ids otherwise we will not be able to add individual items